src/pages/index.js

```diff
@@ -41,70 +55,243 @@ addFormValidator.enableValidation();
 
 const toggleAddButtonState = () => addFormValidator.toggleButtonState();
 
-//Initialize Modals
-const editProfileModal = new PopupWithForm({
-  popupSelector: "#edit-profile-modal",
-  handleFormSubmit: (data) => {
-    userInfo.setUserInfo({ name: data.name, job: data.job });
-  },
-});
+///////////////////////////////////////////////////////////////////////////
+//       Initialize Classses & Modals       //////////////////////////////
+//////////////////////////////////////////////////////////////////////////
 
-const addCardModal = new PopupWithForm({
-  popupSelector: "#add-card-modal",
-  handleFormSubmit: (data) => {
-    const cardElement = createCard({ name: data.title, link: data.url });
-    section.addItem(cardElement);
-    cardForm.reset();
-    toggleAddButtonState();
+// Initialize API //
+const api = new Api({
+  baseUrl: "https://around-api.en.tripleten-services.com/v1",
+  headers: {
+    authorization: "c58369c4-dcd4-4208-8726-c10df21880b5",
+    "Content-Type": "application/json",
   },
 });
 
-// Initialize User Info
+// Initialize User Info //
 const userInfo = new UserInfo({
   nameSelector: ".profile__name",
   jobSelector: ".profile__description",
+  avatarSelector: ".profile__avatar",
 });
 
-// Intialize Section
+// Intialize Section //
 const section = new Section(
   {
-    items: initialCards,
+    items: [],
     renderer: (cardData) => {
-      const cardElement = createCard(cardData);
-      section.addItem(cardElement);
+      section.addItem(cardData);
     },
   },
   ".cards"
 );
 
-section.renderItems();
+console.log("Section instance:", section);
 
-// Modal Event Listeners
-editProfileModal.setEventListeners();
-addCardModal.setEventListeners();
-imagePreviewModal.setEventListeners();
+// Fetch and Render Initial Data//
+api
+  .getAllData()
+  .then(({ user, cards }) => {
+    console.log("Initial user data:", user);
+    console.log("Initial cards data:", cards);
+    const cardElement = cards.map((cardData) => createCard(cardData));
```

Comment:
`createCard` should be used inside method `renderer`, which should be used inside method `renderItems` according to the task

Your code will be like this


```js
  .then(({ user, cards }) => {
    section.renderItems(cards);
    userInfo.setUserInfo(user);

```